packages/addons/service/fd628/package.mk

```diff
@@ -0,0 +1,45 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2018-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="fd628"
+PKG_VERSION="1.0"
+PKG_REV="1"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="https://libreelec.tv"
+PKG_URL=""
+PKG_DEPENDS_TARGET="toolchain"
+PKG_SECTION="service"
+PKG_SHORTDESC="fd628: Kodi service to light up additional icons on devices with FD628 display"
+PKG_LONGDESC="fd628: Kodi service to light up additional icons on devices with FD628 display"
+PKG_TOOLCHAIN="manual"
+
+PKG_IS_ADDON="yes"
+PKG_ADDON_NAME="service.fd628"
```

Comment:
Addon name is then service.fd628 displayed in Kodi - I guess this isn't too helpful. Shouldn't it be named "LCD Display Driver" or something ? Because the user might not know what service.fd628 is ;)

Reply:
It is shown as 'FD628 Service' to the user, I wasn't sure what to call the add-on hence I asked for ideas, this isn't a driver though and is just an accompanying service to light up the additional icons on the display, the display is a crappy calculator/microwave oven VFD type as opposed to an LCD.

How about, 'FD628 VFD Display Service' ?

Reply:
"FD628 Display" is easier to translate, uses less visual space, and avoids anyone having to decide whether it's a driver or service and what that may or may not mean to a user.

Reply:
So what should the name for the package/zip be and which folder should I be placing it in?

Have you any thoughts on the summary/description as well?